feed/utils.js

```diff
@@ -0,0 +1,50 @@
+async function getActivityFeedData(query = {}, nextLink) {
+  let finalUrl =
+    API_BASE_URL + (nextLink || '/logs' + generateActivityFeedParams(query));
+  const res = await fetch(finalUrl, {
+    credentials: 'include',
+    method: 'GET',
+    headers: {
+      'Content-type': 'application/json',
+    },
+  });
+  return await res.json();
+}
+
+function generateActivityFeedParams(query) {
+  let queryParam = 'dev=true&format=feed';
+  if (query?.category) queryParam += getLogTypesFromCategory(query?.category);
+  return `?${queryParam}`;
+}
+
+function getLogTypesFromCategory(category) {
```

Comment:
Consider adding a default case to handle unexpected categories or errors gracefully. It could return an empty string or throw an error for clarity.

Reply:
Done. Miss from my side. Didn't add default case for category.